twiml/voice/connect/virtualagent/virtualagent.7.x.java

```diff
@@ -0,0 +1,19 @@
+import com.twilio.twiml.voice.Connect;
+import com.twilio.twiml.VoiceResponse;
+import com.twilio.twiml.voice.VirtualAgent;
+import com.twilio.twiml.TwiMLException;
+
+
+public class Example {
+    public static void main(String[] args) {
+        VirtualAgent virtualagent = new VirtualAgent.Builder().connectorName("project").statuscallback("https://mycallbackurl.com").build();
```

Comment:
Weird casing too....hmmmm

The ClassName (in PascalCase) should create an instanceObject (in camelCase)

Reply:
I'm not going to stop the approval, but maybe add an issue to TwiML Generator?

Reply:
That's true -- I should actually go test these first 🤦‍♀️ . I'll check out what's been generated.